Core/src/Definitions/Direction.cpp

```diff
@@ -0,0 +1,28 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Definitions/Direction.hpp"
+
+#include <ostream>
+
+std::string Acts::Direction::toString() const {
```

Comment:
Actually, in order to be a bit more efficient, we could return `std::string_view` here, right?